### ISSUE TITLE
I lose wifi hotspot when camera connected to usb3.

### ISSUE THREAD

**Issue by costargazer** [rkaczorek/astroberry-server#228]
Interesting problem.  I have two raspberry pi 4b, one with 8gb, one with 4gb.  I am able to log onto the wifi hotspot remotely, then when I plug my camera into the usb3 on the 8gb model I lose the wifi hotspot.  It doesn't happen with the 4gb model.  I flashed the micro sd card on both with the same img file. Ran the update after the initial install. Searched the issues but did not find anything.
Thank You,
Scott

**Comment by rkaczorek** [comment]
It could be well known issue of USB 3.0 and WiFi 2.5GHz conflict coming from h/w design flaw of RPi 4. If the devices connected to USB are exactly the same, I can only guess that these 2 versions of RPi might differ somehow. What you can do is to change hotspot configuration so it uses 5GHz WiFi instead.

**Comment by costargazer** [comment]
Yes, I was doing some more research. I'm going to try 5Ghz, and maybe a clamp on ferrite choke on the usb cable.
Thank You

**Comment by nystire** [comment]
Did the ferrite choke help?

**Comment by bbillp** [comment]
Keep USB cables away from the Raspberry Pi, USB emits strong radio frequency signals to disrupt WiFi and BlueTooth.    Bill Sent from my iPadOn Jul 31, 2024, at 10:07 AM, nystire ***@***.***> wrote:﻿
Did the ferrite choke help?
